frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -419,7 +356,7 @@ export const AboutResourcePage = ({
           />
           {showAllErrors &&
             hasTitleError &&
-            displayWarningCard(getMissingInputLanguage(title, 'tittel'))}
+            displayWarningCard(getMissingInputLanguageString(title, 'tittel'))}
```

Comment:
I think we need to take a look at this when implementing translations with i18n. :)

Reply:
Yes, I agree